pkg/strategy/production_test.go

```diff
@@ -99,10 +100,18 @@ func TestOptionsArePassed(t *testing.T) {
 	deployments := getDeployments(objs)
 	for _, dep := range deployments {
 		args := dep.Spec.Template.Spec.Containers[0].Args
-		assert.Len(t, args, 3)
+		// Including parameter for sampling config
+		assert.Len(t, args, 4)
+		var escount int
 		for _, arg := range args {
-			assert.Contains(t, arg, "es.")
+			if strings.Contains(arg, "es.") {
+				escount++
+			}
 		}
+		assert.Equal(t, 3, escount)
+		// TODO: Added break as deployments includes both the original and modified
```

Comment:
Not clear why the original and modified deployments are both being returned by `getDeployments(objs)` aibove?

Reply:
Sorry my mistake - were for different deployments :smile: